DiagnosticsExtension/Models/ConnectionStringValidator/ConnectionStringValidationResult.cs

```diff
@@ -8,17 +8,31 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.Json.Serialization;
 using System.Web;
+using System.Web.UI.WebControls;
+using Newtonsoft.Json;
 
 namespace DiagnosticsExtension.Models.ConnectionStringValidator
 {
 
     public class ConnectionStringValidationResult
     {
+        [Newtonsoft.Json.JsonIgnore]
         public ResultStatus? Status;
+        [Newtonsoft.Json.JsonIgnore]
+        public string IdentityType;
+        [JsonProperty("Summary")]
```

Comment:
Can we simply change these properties name instead of overriding them?

Reply:
Yes, that's fair.